data/src/main/java/com/microsoft/azure/kusto/data/ClientImpl.java

```diff
@@ -228,10 +249,10 @@ private CommandType determineCommandType(String command) {
         return CommandType.QUERY;
     }
 
-    private Map<String, String> generateIngestAndCommandHeaders(ClientRequestProperties properties,
-                                                                String clientRequestIdPrefix,
-                                                                String activityTypeSuffix)
-            throws DataServiceException, DataClientException {
+    private Map<String, String> generateIngestAndCommandHeaders(
+            ClientRequestProperties properties,
+            String clientRequestIdPrefix,
+            String activityTypeSuffix) throws DataServiceException, DataClientException {
```

Comment:
I think this *reduces* readability.

Reply:
Again, it's consistent.
If you want it to be consistently like the old style we can discuss this again (though I tried to spark this discussion multiple times and nobody answered).
I don't care what it would be in the end, as long as it's one style.